code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm

```diff
@@ -147,6 +147,44 @@
 		X.use_plasma(plasma_cost)
 	add_cooldown()
 
+// ***************************************
+// *********** Forward Charge
+// ***************************************
+/datum/action/xeno_action/activable/forward_charge
+	name = "Forward Charge"
+	action_icon_state = "charge"
+	mechanics_text = "Charge up to 4 tiles and knockdown any targets in your way."
+	ability_name = "charge"
+	cooldown_timer = DEFENDER_CHARGECOOLDOWN
+	plasma_cost = 80
+
+/datum/action/xeno_action/activable/forward_charge/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!A)
+		return FALSE

```

Comment:
```suggestion
		CRASH("forward_charge/can_use_ability invoked without an atom/A")
```
We should be using `atom/movable/AM` here, though. I know other things don't, but, you know.